crates/meroctl/src/cli.rs

```diff
@@ -173,3 +264,8 @@ where
 {
     serializer.collect_str(&report)
 }
+
+pub struct ConnectionInfo {
+    pub api_url: Url,
+    pub auth_key: Option<String>,
```

Comment:
why not make this a `Keypair`?

```suggestion
    pub auth_key: Option<Keypair>,
```